bootkit/main.go

```diff
@@ -24,6 +24,9 @@ type tinkConfig struct {
 	registry string
 	username string
 	password string
+	certURL string
```

Comment:
I think certURL makes sense here, we can probably infer `registryCertRequired` based on the presence of certURL